app/views/setup/discovery.html.slim

```diff
@@ -0,0 +1,34 @@
+.alert.alert-info.alert-dismissible role="alert"
+  button.close[type="button" data-dismiss="alert" aria-label="Close"]
+    span[aria-hidden="true"]
+      | &times;
+  i.fa.fa-4x.pull-left aria-hidden="true"
+  span
+    | After choosing the master and clicking "Bootstrap cluster" all other nodes will be set to the minion role
+h1 Nodes
+.row
+  .col-xs-12.discovery-control
+    span #{Minion.count} Nodes found
+= form_tag(setup_bootstrap_path, method: "post")
+.nodes-container
+  table.table
+    thead
+      tr
+        th
+          | Id
+        th
+          | Hostname
+        th.text-center
+          | Master
```

Comment:
Andreas specifically said that we will have this information and it should be included. Please do so.